Makefile

```diff
@@ -0,0 +1,9 @@
+planout:
+	lua buildresources/pack.lua src/ > dist/planout.lua
+
+install:
+	brew install lua
```

Comment:
Might want to note that this is OS X specific, and/or support other things? Also would be useful to include a README.md explaining how this (and other things) work.